web/gridt_server/tests/unit/test_schemas.py

```diff
@@ -9,7 +9,6 @@
 )
 
 from freezegun import freeze_time
-from gridt_server.models.user import User
 
 
 class SchemasTest(BaseTest):
```

Comment:
We should fix this before moving on. We should keep tests that do not use models!